liquidprompt

```diff
@@ -224,6 +224,7 @@ __lp_source_config() {
     LP_ENABLE_COLOR=${LP_ENABLE_COLOR:-1}
     LP_ENABLE_ERROR=${LP_ENABLE_ERROR:-1}
     LP_ENABLE_DIRSTACK=${LP_ENABLE_DIRSTACK:-0}
+    LP_ENABLE_KUBECONTEXT=${LP_ENABLE_KUBECONTEXT:-1}
```

Comment:
This should be disabled by default, as unlike other development environments (E.g. in Python where you need to `. venv/bin/activate`) a `kubectl` context is always on, and it is not possible (or at least not intended) to have no context at all. Meaning if you have `kubectl` installed, a context will _always_ show up in the prompt.